ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/SFLOW_TESTS.yml

```diff
@@ -5,6 +5,21 @@ fabric_name: SFLOW_TESTS
 
 mgmt_gateway: 192.168.0.1
 
+default_mgmt_method: oob
+
+sflow_settings:
+  destinations:
+    - destination: 10.10.10.10
+      vrf: use_mgmt_interface_vrf
+    - destination: 10.10.10.11
+    - destination: 10.10.10.12
+      vrf: sflowvrf
+      port: 1234
+  structured_config:
+    vrfs:
+      - name: sflowvrf
+        source_interface: Loopback321
```

Comment:
looking at this - would it not be easier to have the `structured_config` inside the destinations list to apply on the VRF directly? or simply adding source_interface at the destinations level? is there other use case for structured config here?

Reply:
For this specific config it could be done in other ways, but the whole point of structured_config is that we can put anything in there :). The area contains several things we don't support here (yet).

Reply:
I am planning a follow on PR with a single boolean knob to build the streaming via terminattr. But there are some complexity when it comes to L2 switches. We will need to create a loopback interface in the default VRF, and that interface must have an IP address. So the second we add that, the requests for customizing the assigned loopback number and IP will also arise :).